discovery-provider/prod.env

```diff
@@ -1,5 +1,6 @@
 # DO NOT MODIFY these values, use `audius-cli get-config discovery-provider` for accurate values
 
+audius_openresty_enable=true
```

Comment:
Why is this not enabled by default?

Reply:
Follow up: add to stage.env, as well as creator-node/[prod.env|stage.env].

Reply:
Don't know here.

@cheran-senthil is this ok?

Reply:
is because openresty stuff requires `audius_discprov_url` which is usually not required and it used to be buggy so we only used on audius controlled nodes

Reply:
This is not safe

Reply:
good callout - i'm not authority on discovery but makes sense to separate enabling by default from this work - lets remove this flag here and that way we can manually do it on our nodes and make sure things look good